src/eureka/S3_data_reduction/plots_s3.py

```diff
@@ -95,8 +95,8 @@ def image_and_background(data, meta, log, m):
     xmin, xmax = data.flux.x.min().values, data.flux.x.max().values
     ymin, ymax = data.flux.y.min().values, data.flux.y.max().values
 
-    # FINDME: Max calculation is broken and I can't figure out how to fix it
-    # vmax = np.nanmax(~data.mask.values*subdata)/10
+    # Commented out vmax calculation is sensitive to unflagged hot pixels
+    # vmax = np.ma.max(np.ma.masked_invalid(subdata))/40
```

Comment:
I don't think you should need the np.ma.masked_invalid since it is already a masked array and data.mask should already mask the bad pixels. But it won't do any harm either